SayTheirNames.xcodeproj/xcshareddata/xcschemes/SayTheirNames.xcscheme

```diff
@@ -73,6 +73,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "es"
```

Comment:
This might be an unintended change caused when setting the Application Language to Spanish in the SayTheirNames scheme. I'd suggest setting it again to System Language before merging.

Reply:
@Angel5215 good catch, I will create a separate PR for this.

Reply:
@Angel5215 oh good catch! I forgot to change it back before creating the PR